VogtPayroll3/Display.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll3
+{
+    class Display
+    {
+
+        public Dictionary<int, Employee> PrintMenu()
+        {
+            Employee emp = new Employee("holder", "holder", 1, 1, 1.0m);
+            char option;
+            var empID = 0;
+            List<Employee> empList = new List<Employee>();
+            PayrollManager payrollManager = new PayrollManager();
+            PayrollConsoleReader payrollConsoleReader = new PayrollConsoleReader();
+            Dictionary<int, Employee> dictionary = new Dictionary<int, Employee>();
+
+            Console.WriteLine("c. Create a new employee");
+            Console.WriteLine("h. Change an employee");
+            Console.WriteLine("u. Update an employee");
+            Console.WriteLine("d. Delete records");
+            Console.WriteLine("q. quit the program");
+            option = Console.ReadKey().KeyChar;
+
+            while (option != 'q')
+            {
+                switch (option)
+                {
+                    case 'q':
+                        System.Environment.Exit(0);
+                        break;
+                    case 'c':
+                        Console.WriteLine("");
+
+                        emp = emp.CreateAnEmployee();
+                        dictionary = payrollManager.AddEmployeeToDictionary(emp);
```

Comment:
Good! These are nice and clean switch case statements.

And good -- youre creating the employee here then pushing to the manager.

Though see my note above about using the employee to create another employee.